File: 3455-minimum-length-of-string-after-operations/minimum-length-of-string-after-operations.cpp
class Solution {
public:
    int minimumLength(string s) {
        vector<int> fre(26,0);
        for (auto& it : s) {
            fre[it - 'a']++;
        }
        
        int ans = 0;
        for (auto& it : fre) {
            if (it % 2)
                ans += 1;
            else
                ans += it?2:0;

        }
        return ans;
    }
};

File: 3455-minimum-length-of-string-after-operations/minimum-length-of-string-after-operations.go
func minimumLength(s string) int {
    fre := make([]int , 26)
    for _ , c := range s{
        fre[c-'a']++
    }
    ans:=0
    for _ , occ := range fre{
        if occ%2!=0 {
            ans++
        } else {
            if occ > 0 {
                ans+=2
            }
        }
    }
    return ans
}